### ISSUE TITLE
the loss is Nan

### ISSUE THREAD

**Issue by JinMana** [RaidasGrisk/tf2-crnn#3]
hello, when I train my own dataset, the training loss is always Nan, how to resolve this problem?
